10_install_server/snippets/zypper_segfault.txt.snippet

```diff
@@ -0,0 +1,16 @@
+[WARNING]
+
+====
+
+You might not be able to install Rudder rpm files locally with Zypper
+(for example with 'zypper install rudder-agent-version.release-1.SLES.11.x86_64.rpm'),
+due to a bug (https://bugzilla.novell.com/show_bug.cgi?id=889363[bnc#929483] on SuSE bugtracker) in its RPM headers parsing causing a segmentation fault.
+You can either:
+
+* Install the packages directly from the repository, as described below
+* Upgrade your libzypp package to a version including the fix provided by SuSE (upgrade for http://download.novell.com/Download?buildid=42q80kCaQxE[SLES11SP3] and
+  for http://download.novell.com/Download?buildid=0cI2ELIttrI[SLES12]) 
+* Use the rpm command to install packages locally
```

Comment:
Could you provide an example as above for this one? Something like "(for example with 'rpm -i rudder-agent-version.release-1.SLES.11.x86_64.rpm')"